src/config.ts

```diff
@@ -209,4 +209,6 @@ export default {
   VORTEX_API_BASE,
   VORTEX_APP_ID,
   VORTEX_TOKEN,
+  GRAPHQL_UPLOAD_MAX_FILE_SIZE_IN_BYTES: 5 * 1000 * 1000, // 5 MB
```

Comment:
**suggestion**: It looks like the recommended average size when searching for images against the TinEye API is 100kb ([source](https://services.tineye.com/developers/mobileengine/api_reference/)). What to you think of reducing this default to 300kb to start? 5MB sounds pretty big.

Reply:
> What to you think of reducing this default to 300kb to start?

I have nothing against it 😅